tests/ndimage/footprint_filter_test.py

```diff
@@ -0,0 +1,192 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+import numpy as np
+import scipp as sc
+import scipy.ndimage
+from scipp.ndimage import (generic_filter, maximum_filter, median_filter,
+                           minimum_filter, percentile_filter, rank_filter)
+import pytest
+
+filters = (generic_filter, maximum_filter, median_filter, minimum_filter,
+           percentile_filter, rank_filter)
+
+
```

Comment:
Very thorough :+1:

Reply:
Actually I am cheating, since not all tests are using the fixture that gets all filters.

Reply:
I meant in general. But apparently we both missed a case